db/src/lib.rs

```diff
@@ -35,6 +42,8 @@ mod values;
 
 use edn::symbols;
 
+pub const SQLITE_MAX_VARIABLE_NUMBER: usize = 999;
```

Comment:
This is a compile-time option in SQLite. We should upstream a patch to rusqlite.